src/services/workflow_orchestrator.py

```
@@ -38,7 +60,9 @@ def __init__(self):
 
     def run(self, job_description: str):
         print(f"---ORCHESTRATOR: Starting workflow...---")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)